CMakeLists.txt

```diff
@@ -139,6 +139,12 @@ if(BUILD_TESTING)
     WORKING_DIRECTORY "$<TARGET_FILE_DIR:test_logging_long_messages>"
     TIMEOUT 10)
 
+  ament_add_nose_test(test_logging_output_format
+    "test/test_logging_output_format.py"
+    APPEND_LIBRARY_DIRS ${extra_lib_dirs}
+    WORKING_DIRECTORY "$<TARGET_FILE_DIR:test_logging_long_messages>"
```

Comment:
this test re-uses the long_message test binary and just modifies the output format from an environment variable. In the custom output, the long message is output twice, to test both dynamic allocation and re-allocation.